easybuild/easyconfigs/i/InterProScan/InterProScan-5.62-94.0-with_data.eb

```diff
@@ -0,0 +1,70 @@
+easyblock = 'Tarball'
+
+name = 'InterProScan'
+version = '5.62-94.0'
+versionsuffix = '-with_data'
+
+homepage = 'https://interproscan-docs.readthedocs.io/'
+# Also https://github.com/ebi-pf-team/interproscan
+description = """
+InterPro is a database which integrates together predictive information about
+proteins’ function from a number of partner resources, giving an overview of the
+families that a protein belongs to and the domains and sites it contains.
+
+Users who have novel nucleotide or protein sequences that they wish to
+functionally characterise can use the software package InterProScan to run the
+scanning algorithms from the InterPro database in an integrated way.  Sequences
+are submitted in FASTA format.  Matches are then calculated against all of the
+required member database’s signatures and the results are then output in a
+variety of formats."""
+# Licence: Apache License 2.0.
+
+toolchain = SYSTEM
```

Comment:
this should not be at `SYSTEM` level if it has runtime dependencies on `GCCcore`

Reply:
@jfgrimm OK, my train of thought was that the InterProScan build itself has no direct dependency on GCCcore, therefore it did not rely on a toolchain.  

But it does depend on other packages that themselves depend on GCCcore.  Does that create a toolchain dependency for InterProScan, in your view?

Reply:
yes, because one way or another you are pulling in the toolchain at runtime. As such, marking the module as `SYSTEM` incorrectly suggests that it isn't pinned to a particular toolchain